src/storage/mod.rs

```diff
@@ -4559,6 +4576,25 @@ mod tests {
         let mut range = KeyRange::default();
         range.set_start_key(b"r\0a".to_vec());
         range.set_end_key(b"r\0z".to_vec());
+        assert_eq!(
+            (checksum, total_kvs, total_bytes),
+            block_on(storage.raw_checksum(ctx.clone(), ChecksumAlgorithm::Crc64Xor, vec![range]))
+                .unwrap(),
+        );
+
+        let mut range = KeyRange::default();
```

Comment:
The test case [here](https://github.com/tikv/tikv/blob/50e0cf4ee720a36ce62a2d80c341cf48533e2977/tests/integrations/storage/test_storage.rs#L1102) may also need to be changed.

Reply:
This case is ok, as it juse use the prefix to be the start key. The test case added is a negative case to prevent the encoded key as the input.

Reply:
The input range is left closed and right open interval. If changed to `b"r\0f"`, whatever the key is encoded or not, the entry `(b"r\0f", b"ff")` will not be included in checksum.

Reply:
Changed the first key to `b"r\0a0"` to simplify the case.